src/main/java/org/broadinstitute/hellbender/engine/MultiVariantWalker.java

```diff
@@ -63,7 +63,8 @@ protected MultiVariantInputArgumentCollection getMultiVariantInputArgumentCollec
     protected void initializeDrivingVariants() {
         multiVariantInputArgumentCollection.getDrivingVariantPaths().stream().forEach(
                 f -> {
-                    FeatureInput<VariantContext> featureInput = new FeatureInput<>(f);
+                    // use getRawInputString() on the original GATKPath here to ensure that we propagate any argument tags
+                    FeatureInput<VariantContext> featureInput = new FeatureInput<>(f.getRawInputString());
```

Comment:
I added a copy constructor to GATKPath, along with a `FeatureInput(GATKPath)` constructor, and used that here and below.

`FeatureInput` can be dramatically simplified once there are no more call sites that use the constructors that take Strings (ie., it no longer needs to have overloads that take tag maps, or even names, since they can/should be propagated though GATKPath). I started going down that path but it quickly cascaded to lots of changes, so for now I'm doing the minimum and reserving those changes for a separate PR.